### PR TITLE
added option to pass segment number through unpackFrame_UAVSAR

### PR DESCRIPTION
**Pull request by EJFielding**
Modified `stripmapStack` `prepareUAVSAR_coregStack.py` and `unpackFrame_UAVSAR.py` to pass requested segment number to the `UAVSAR_stack` sensor object. Default is still segment 1, but this allows processing other segments of a multi-segment UAVSAR SLC stack.

### CODE REVIEW

**Comment by EJFielding** [comment]
Addresses https://github.com/isce-framework/isce2/issues/170

File: contrib/stack/stripmapStack/unpackFrame_UAVSAR.py [hunk 1]
```diff
@@ -14,17 +15,19 @@ def cmdLineParse():
...
     '''
 
-    parser = argparse.ArgumentParser(description='Unpack CSK SLC data and store metadata in pickle file.')
+    parser = argparse.ArgumentParser(description='Unpack UAVSAR SLC data and store metadata in pickle file.')
     parser.add_argument('-i','--input', dest='h5dir', type=str,
-            required=True, help='Input CSK directory')
+            required=True, help='Input UAVSAR directory')
     parser.add_argument('-d','--dop_file', dest='dopFile', type=str,
             default=None, help='Doppler file')
+    parser.add_argument('-s','--segment', dest='stackSegment', type=str,
```
Comment (piyushrpt) [inline contrib/stack/stripmapStack/unpackFrame_UAVSAR.py:23]: The type here should be an int. Else a string will be passed through for any value other than the default raising an error.

**Comment by piyushrpt** [comment]
@nicodalse - could you test these changes and confirm this works?

**Review by hfattahi — APPROVED** [review]
LGTM. Please rebase and should be good to go.

**Review by yunjunz — APPROVED** [review]

**Comment by EJFielding** [comment]
> LGTM. Please rebase and should be good to go.

@hfattahi What git command or commands is best to rebase this branch?

**Comment by rtburns-jpl** [comment]
No need to rebase, you don't have any conflicts with the main branch. I'll just go ahead and merge.

**Comment by EJFielding** [comment]
Thanks @rtburns-jpl !

**Comment by rtburns-jpl** [comment]
@piyushrpt Merge job is failing to login to docker hub to upload the image, did you update the docker hub credentials recently? If so we need to update https://app.circleci.com/settings/project/github/isce-framework/isce2/environment-variables

**Comment by piyushrpt** [comment]
Oops ... yes. Moved everything to a password manager and randomized passwords this weekend. Maybe we should start using a better way of using credentials - moving fwd with isce3. I have temporarily updated the password - but might break again automatically in a month.

**Comment by rtburns-jpl** [comment]
Great, thanks. Happy to take over managing our docker credentials, shouldn't have to be your responsibility. Good on you for using a password manager :)

**Comment by piyushrpt** [comment]
Feel free to switch it over to another registered user and password by modifying the env variables.

**Pull request merged**